docs/contributing/opensauced-writers-guide/how-to-add-your-post.md

```diff
@@ -0,0 +1,98 @@
+---
+id: writers-guide
+title: "OpenSauced Writers Guide"
+sidebar_label: "OpenSauced Writers Guide"
+keywords: 
+- "writers guide" 
+- "open source writers guide" 
+- "open source writer" 
+---
+
+Welcome to our Writers Contributor Guide. This will walk you through the steps of how to add your blog post to our community resources using the docs repository.
+
+## Repeat Authors: Adding to `authors.yml`
+
+For regular blog post authors, we should declare those authors globally in the configuration file:
```

Comment:
[proofreading]: Could be a bit more clearer
```suggestion
For regular authors, we should declare them globally in the configuration file:
```

Reply:
I think it's fine the way it is.